README.md

```diff
@@ -233,13 +233,6 @@ $ dcrwallet --create
 $ dcrwallet
 ```
 
-- Get the master pubkey from the default account.  This will be used for
```

Comment:
I think we should **update** rather than **remove** this piece of documentation. We can explain how the key is now derived automatically from the `default` account